### ISSUE TITLE
Error when we connect Vector Tiles Reader with Tiles.json

### ISSUE THREAD

**Issue by tanish-creator** [geometalab/Vector-Tiles-Reader-QGIS-Plugin#285]
When I connect to Tiles.json URL in QGIS (3.4.14-Madeira) Vector Tiles Reader plugin (3.2.2)(http://localhost/VectorTileService/tiles.json), getting
Creating reader failed: (, RuntimeError('Loading error: Resource not found.\n\nURL incorrect?'), ) error.

Attached the snapshots and log files...
[log.txt](https://github.com/geometalab/Vector-Tiles-Reader-QGIS-Plugin/files/5275995/log.txt)
![QGIS2](https://user-images.githubusercontent.com/70086705/94145659-834f9d00-fe90-11ea-92a7-77066b054300.PNG)
![QGIS1](https://user-images.githubusercontent.com/70086705/94145665-8480ca00-fe90-11ea-8ccd-00925d66556a.PNG)

**Comment by mnboos** [comment]
Hi and thanks for reporting this issue. 

Can you open the exact same URL in your browser?

**Comment by tanish-creator** [comment]
Yes. I am able to open the URL in the browser. Also, If I place the JSON file on the physical drive and access i.e., C:\Users\rmaddu\Desktop\tiles.json, no issues. 

This issue is with accessing the URL directly. How to resolve this? the logfile is not giving many clues...no information in EventViewer also. Please help us to resolve this issue.

Thanks for your time.

**Comment by mnboos** [comment]
Try http://127.0.0.1/VectorTileService/tiles.json please

**Comment by tanish-creator** [comment]
Hi, getting the same error with http://127.0.0.1/VectorTileService/tiles.json.

**Comment by tanish-creator** [comment]
Please let me know if you need any further information.
